package.json

```diff
@@ -19,15 +21,18 @@
     "test",
     "suite"
   ],
-  "author": "digital_bazaar",
+  "author": {
+    "name": "Digital Bazaar, Inc.",
+    "email": "support@digitalbazaar.com",
+    "url": "https://digitalbazaar.com/"
+  },
   "license": "MIT",
```

Comment:
Is this the appropriate license?  Elsewhere, we use this license file, and don't have a `license` property in the package file: https://github.com/w3c-dvcg/security-vocab/blob/master/LICENSE.md